index.html

```diff
@@ -2,9 +2,45 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <link
```

Comment:
no need for placing attributes in new lines

Reply:
Corrected